.github/workflows/tests.yml

```diff
@@ -1,8 +1,7 @@
 name: GitHub Actions CI
 on:
-  push:
-    branches: master
-  pull_request: []
+  - push
```

Comment:
Don't want this because it'll duplicate jobs on non-fork branch pushes.

Reply:
I'm trying to get the output of

```
curl -H 'Accept: application/vnd.github.antiope-preview+json' https://api.github.com/repos/reitermarkus/brew/commits/automerge/check-runs
```

Reply:
Sorry, you've lost me here.

Reply:
I'm just experimenting.

If the action doesn't run on `push`, the Check Runs API returns an empty array when I make the request above.

Reply:
Gotcha, I see. Cool to experiment just don't merge this change into Homebrew/brew, ta 👍